app/components/BuyNBTC.tsx

```diff
@@ -13,6 +13,31 @@ import { suiToMist } from "~/util/util";
 import { useToast } from "~/hooks/use-toast";
 import { pricePerNBTCInSUI } from "~/constant";
 
+const BUY_NBTC_INSTRUCTIONS = [
+	"Click on Connect Sui Wallet button, if not already connected.",
+	"Use the Slush wallet.",
+	`Click "profile" → network → make sure you select testnet.`,
+	`Click "profile" → scroll down to the "About" section → click "Request Sui Token".`,
```

Comment:
maybe we should add a link to the testnet faucet for users that do not use slush?